### PR TITLE
Update list of Action types on Alerts

### LINKED ISSUE #1998: [Request] Update list of Action types on Alerts
## Description

More integrations are now available for defining actions on alerts. Screenshots in the following pages need to be updated with newer ones:

- https://www.elastic.co/guide/en/observability/current/logs-threshold-alert.html
- https://www.elastic.co/guide/en/observability/current/infrastructure-threshold-alert.html
- https://www.elastic.co/guide/en/observability/current/metrics-threshold-alert.html
- https://www.elastic.co/guide/en/observability/current/monitor-status-alert.html
- https://www.elastic.co/guide/en/observability/current/tls-certificate-alert.html
- https://www.elastic.co/guide/en/observability/current/duration-anomaly-alert.html

## Collaboration

- The product team will provide the initial content and the docs team will edit / review
- Contact Person: @benakansara 

## Suggested Target Release

8.4.0

### PR DESCRIPTION
**Pull request by colleenmcginnis**
Closes #1998

Updates screenshots of action types and updates a few outdated UI references.

cc @emma-raffenne

### CODE REVIEW

**Review by bmorelli25 — APPROVED** [review]
Those arrows look great too!

**Comment by dedemorton** [comment]
>Those arrows look great too!
Yas! Let's do this with all the observability docs. Much better for accessibility.

**Comment by bmorelli25** [comment]
Something for the linter, perhaps? If regex matches `** > **`, recommend `->` instead?

**Comment by dedemorton** [comment]
> If regex matches ** > **, recommend -> instead?

Good idea...only crinkle is that I use `**UI control > UI control > UI control**` (though I'm sure we could come up with a regex for that too).

**Comment by bmorelli25** [comment]
You're fast! I edited out my example a minute after posting when I realized I'm the only one that follows the `** > **` pattern :)

**Review by benakansara — APPROVED** [review]
LGTM.

File: docs/en/observability/infrastructure-threshold-alert.asciidoc [hunk 1]
```diff
@@ -8,8 +8,8 @@ resource or a group of resources within your infrastructure.
 Additionally, each rule can be defined using multiple
 conditions that combine metrics and thresholds to create precise notifications and reduce false positives.
 
-. To access this page, go to *{observability} > Metrics*.
-. On the *Inventory* page or the *Metrics Explorer* page, click *Alerts > Infrastructure*.
+. To access this page, go to **{observability}** -> **Metrics**.
```
Comment (benakansara) [inline docs/en/observability/infrastructure-threshold-alert.asciidoc:11]: It looks like the name of menu item is changed. There is no **Metrics** item in menu, instead it is **Infrastructure**. 
Screenshot 2022-08-16 at 13 32 02

File: docs/en/observability/metrics-threshold-alert.asciidoc [hunk 2]
```diff
@@ -1,25 +1,25 @@
...
 you can create a threshold rule to notify you when a metric has reached or exceeded a value for a specific
 time period.
 
 Additionally, each rule can be defined using multiple
 conditions that combine metrics and thresholds to create precise notifications.
 
-. To access this page, go to *{observability} > Metrics*.
-. On the *Inventory* page or the *Metrics Explorer* page, click *Alerts > Metrics*.
-. Select *Create threshold alert*.
+. To access this page, go to **{observability}** -> **Metrics**.
```
Comment (benakansara) [inline docs/en/observability/metrics-threshold-alert.asciidoc:11]: Same as first comment. The menu item is now **Infrastructure**.

**Review by benakansara — COMMENTED** [review]
I have left comments regarding UI references in doc.

File: docs/en/observability/infrastructure-threshold-alert.asciidoc (hunk 1 above)
Comment (colleenmcginnis) [reply docs/en/observability/infrastructure-threshold-alert.asciidoc:11]: Thanks for letting us know! I'll address this separately. See https://github.com/elastic/observability-docs/issues/2097.

**Pull request merged**